force_wfmanager/ui/setup/process/process_view.py

```diff
@@ -0,0 +1,100 @@
+from traits.api import (
+    HasTraits, Instance, List, on_trait_change, Bool, Event, Unicode
+)
+from force_wfmanager.utils.variable_names_registry import (
+    VariableNamesRegistry
+)
+from .execution_layer_view import ExecutionLayerView
+from force_bdss.api import Workflow
+
+
+class ProcessView(HasTraits):
```

Comment:
This new object contains the list of `ExecutionLayers` defined in the `Workflow`. It could also have its own traits view displaying template `Workflows` or a graphical representation of the current `Workflow` structure in the future. It possess the same hierarchy level as the `MCOView`.